scripts/data/gen-protoc/readme.md

```diff
@@ -0,0 +1,87 @@
+# Gen protoc
```

Comment:
I added this readme to explain what I think should be done later for protoc.
Hopefully the readme explains the situation well enough.